api/client-server/key_backup.yaml

```
@@ -0,0 +1,761 @@
+          schema:
+            type: object
+            properties:
+              algorithm:
+                description: |-
+                  The algorithm used for storing backups.  Must be the same as
+                  the algorithm currently used by the backup.
+                type: string
+                enum: ["m.megolm_backup.v1.curve25519-aes-sha2"]
+                example: "m.megolm_backup.v1.curve25519-aes-sha2"
+              auth_data:
+                description: |-
+                  Algorithm-dependent data. See the documentation for the backup
+                  algorithms in `Server-side key backups`_ for more information on the
+                  expected format of the data.
+                type: object
+                example: {
+                    "public_key": "abcdefg",
+                    "signatures": {
+                      "@alice:example.org": {
+                        "ed25519:deviceid": "signature"
+                      }
+                    }
+                  }
+              version:
+                description: |-
+                  The backup version.  If present, must be the same as the
+                  version in the path parameter.
+                type: string
+                example: "1"
+            required:
+              - algorithm
+              - auth_data
+      responses:
+        200:
+          description: The update succeeded
+          schema:
+            type: object
+            properties: {}
+        404:
+          description: The backup specified does not exist
+          examples:
+            application/json: {
+                "errcode": "M_NOT_FOUND",
+                "error": "Unknown backup version"
+              }
+          schema:
+            "$ref": "definitions/errors/error.yaml"
+      tags:
+        - End-to-end encryption
+  "/room_keys/keys/{roomId}/{sessionId}":
+    put:
+      summary: Store a key in the backup
+      description: |-
+        Store a key in the backup.
+      operationId: postRoomKeysKeyRoomIdSessionId
+      security:
+        - accessToken: []
+      parameters:
+        - in: query
+          type: string
+          name: version
+          description: |-
+            The backup in which to store the key. Must be the current backup.
+          required: true
+          x-example: "1"
+        - in: path
+          type: string
+          name: roomId
+          description: The ID of the room that the key is for.
+          required: true
+          x-example: "!roomid:example.org"
+        - in: path
+          type: string
+          name: sessionId
+          description: The ID of the megolm session that the key is for.
+          required: true
+          x-example: "sessionid"
+        - in: body
+          name: data
+          description: "The key data"
+          schema:
+            "$ref": "definitions/key_backup_data.yaml"
+      responses:
+        200:
+          description: The update succeeded
```

```suggestion
          description: The update succeeded.
```


... and I think you get the idea :p